tests/emukit/quadrature/test_measures.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import numpy as np
+import pytest
+
+from emukit.quadrature.kernels.integration_measures import UniformMeasure, IsotropicGaussianMeasure
+
+
+def test_uniform_measure_shapes():
+
+    N = 5
+    bounds = [(-1, 1), (0, 2), (1.3, 5.)]
+    D = len(bounds)
+    x = np.reshape(np.random.randn(D * N), [N, D])
+
+    measure = UniformMeasure(bounds)
+
+    bounds = measure.get_box()
+    assert len(bounds) == D
+    assert len(bounds[0]) == 2
+
+    res = measure.compute_density(x)
+    assert res.shape == (N,)
+
+
+def test_uniform_measure_wrong_bounds():
+    bounds = [(-1, 1), (3, 2), (1.3, 5.)]
+
+    with pytest.raises(ValueError):
+        UniformMeasure(bounds)
+
+
+def test_iso_gauss_measure_shapes():
+    D = 4
+    N = 5
+    x = np.reshape(np.random.randn(D * N), [N, D])
+    measure = IsotropicGaussianMeasure(mean=np.ones(D), variance=1.)
+
+    bounds = measure.get_box()
+    assert len(bounds) == D
+    assert len(bounds[0]) == 2
+    assert measure.dim == D
+
+    res = measure.compute_density(x)
+    assert res.shape == (N, )
+
+
+def test_iso_gauss_measure_invalid_input():
+    variance = -2.
+
+    with pytest.raises(ValueError):
```

Comment:
Glad to see tests for errors! Would be great to have more of these, as you definitely have more failure cases for invalid input in the code

Reply:
I added more tests